### DISCUSSION TITLE
Admin page takes 4.5 seconds to load

### DISCUSSION THREAD

**Discussion by SophiaAtkinson** [YOURLS/YOURLS#3447] [General]
For the prerequisite im running YOURLS 1.9.1
MySQL 8.0.14
PHP 8.0.14
This only started happening after the release 1.9
It doesn't really bother me but it's just confusing.

Here is a chrome network debug for reference.
![](https://linustechtips.xyz/nirO5/yaxUxOKu76.png/raw)

If any of you fine community folks know the problem a response would be great.

Cheers.

**Comment by dgw** [comment]
There's no information here to even speculate.

How big is the DB (10/100/1000/10000/100000/1000000 links)?

Is this server running on a Pi Zero, dual Xeon, or something in between?

Chrome or any other browser won't be able to tell you what's happening on the server; as you see, it will just show that loading the page was slow _for some reason_. You need [debug mode](https://docs.yourls.org/development/debugging.html#yourls-debug) or some kind of [performance profiler](https://xdebug.org/docs/profiler) to figure out _what_ was slow.

That it started after updating to 1.9 could be meaningful, but it might not—for example if PHP or MySQL were updated at the same time.

**Reply by SophiaAtkinson** [reply]
The DB has ~900 links, It's running on a 6-core Xeon VM

I turned on Debug mode and this was the following output
`Connected to database sop.wtf on ccsmdb-scrumpy.internal.satki.net
SQL perform: SELECT option_name, option_value FROM yourls_options WHERE 1=1 (0.00022 s)
SQL perform: SELECT COUNT(keyword) as count, SUM(clicks) as sum FROM `yourls_url` WHERE 1=1  (0.00079 s)
cURL error 28: Connection timed out after 3001 milliseconds (POST on https://api.yourls.org/core/version/1.1/)
SQL perform: UPDATE yourls_options SET option_value = 'O:8:"stdClass":4:{s:15:"failed_attempts";i:636;s:12:"last_attempt";i:1666957873;s:11:"last_result";s:0:"";s:15:"version_checked";s:3:"1.9";}' WHERE option_name = 'core_version_checks' (0.00558 s)
Version check failed: cURL error 28: Connection timed out after 3001 milliseconds (POST on https://api.yourls.org/core/version/1.1/)`

**Reply by SophiaAtkinson** [reply]
Disregard this all. It turns out my proxy went offline, so that's why it was failing to curl https://api.yourls.org/core/version/1.1/

**Reply by ozh** [reply]
Actually that's interesting. We should have something parallel and non blocking to fetch data from api.yourls.org.
